certified-connectors/WordLift GraphQL/Readme.md

```diff
@@ -0,0 +1,15 @@
+## WordLift GraphQL Connector
+
```

Comment:
Hi @ziodave, 

Please update readme.md file with all the required sections. Please refer the template from the below path.

Reply:
Hi @vmanoharas, I updated it.